modules/util/path.go

```diff
@@ -199,3 +198,9 @@ func HomeDir() (home string, err error) {
 
 	return home, nil
 }
+
+// CommonSkipDir will check a provided name to see if it represents directory that should not be watched
+func CommonSkipDir(name string) bool {
+	// Check for Mac's .DS_Store entries
+	return name == ".DS_Store"
+}
```

Comment:
I originally meant something like
```suggestion
func IsMacDSStoreDir(file fs.FileInfo) bool {
	// Check for Mac's .DS_Store entries
	return file.IsDir() && file.Name() == ".DS_Store"
}
```

Reply:
Well that would be fine if we were passing FileInfos but we're not.

We have a FileInfo and a DirEntry.

They don't share a common interface 

So then you're forced to create a new interface to cast against

Then of course we think a bit more and realise it's only Dirs that this is appropriate for - as fs.SkipDir is NOT appropriate for files - so we still have to check if it is a Dir in separate places.

Then, we consider and think a bit more - perhaps it would be appropriate to have a common function - which at the moment only skips the annoying mac directory but probably we should actually skip all "dotdirs" too in future.

Reply:
Oh.
That's of course not good. Yep, then my whole suggestion doesn't really make sense.